test/built-ins/Array/prototype/every/15.4.4.16-3-17.js

```diff
@@ -17,7 +17,12 @@ function callbackfn2(val, idx, obj) {
   return val > 11;
 }
 
-var obj = { 0: 12, 1: 11, 2: 9, length: "0002.00" };
+var obj = {
+  0: 12,
+  1: 11,
+  2: 9,
+  length: "0002.00"
+};
```

Comment:
A lot of these changes are unnecessary, it creates a longer diff.

It's ok for now but I wonder if jsbeautify is the right tool for the next batches.